.github/workflows/trivy.yml

```diff
@@ -0,0 +1,29 @@
+name: trivy
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  trivy:
+    name: trivy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.24.0
+        with:
+          scan-type: 'fs'
+          scan-ref: 'ops/'
```

Comment:
is `scan-ref` indicating the directory of our project we are scanning? (based on my understanding of the [docs here](https://github.com/aquasecurity/trivy-action?tab=readme-ov-file#inputs)) -- do we only want to scan the `ops/` repo or the entire project?

Reply:
Got it!! Thank you for explaining! :D